_data/overviews.yml

```diff
@@ -86,6 +101,12 @@
       icon: tasks
       url: "contributors/index.html"
       description: "Lists all the tools that library authors should setup to publish and document their libraries."
+    - title: "Migration from Scala 2 to Scala 3"
```

Comment:
I don’t think the migration guide should belong to the category “Library Author Guide”, because it targets application programmers too. I think with the current categories the one that would work the best is “Language”, what do you think?

Reply:
I agree - I've moved some other titles within that category also